pom.xml

```diff
@@ -2,81 +2,82 @@
   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
                       http://maven.apache.org/xsd/maven-4.0.0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.untamedears</groupId>
-  <artifactId>PrisonPearl</artifactId>
-  <packaging>jar</packaging>
-  <version>2.4.1</version>
-  <name>PrisonPearl</name>
-  <url>https://github.com/Civcraft/PrisonPearl</url>
-    
-  <properties>
-    <maven.compiler.source>1.6</maven.compiler.source>
-    <maven.compiler.target>1.6</maven.compiler.target>
-  </properties>
-  
-  <build>
-    <sourceDirectory>${basedir}/src</sourceDirectory>
-    
-    <resources>
-      <resource>
-        <directory>${basedir}</directory>
-        <includes>
-          <include>*.yml</include>
-          <include>LICENSE.txt</include>
-        </includes>
-      </resource>
-    </resources>
-  </build>
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>com.untamedears</groupId>
+	<artifactId>PrisonPearl</artifactId>
+	<packaging>jar</packaging>
+	<version>2.4.2</version>
+	<name>PrisonPearl</name>
+	<url>https://github.com/Civcraft/PrisonPearl</url>
 
-  <dependencies>
-    <dependency>
-      <groupId>org.spigotmc</groupId>
-      <artifactId>Spigot1.8.3</artifactId>
-      <version>1.8.3</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>vg.civcraft.mc.namelayer</groupId>
-      <artifactId>NameLayer</artifactId>
-      <version>2.1.1</version>
-      <scope>provided</scope>
-    </dependency>
-	<dependency>
-      <groupId>vg.civcraft.mc.mercury</groupId>
-      <artifactId>Mercury</artifactId>
-      <version>1.0.1-11</version>
-      <scope>provided</scope>
-    </dependency>
-    <dependency>
-      <groupId>com.wimbli</groupId>
-      <artifactId>WorldBorder</artifactId>
-      <version>1.8.3</version>
-      <scope>provided</scope>
-    </dependency>
-	<dependency>
-      <groupId>com.trc202</groupId>
-      <artifactId>Combattag</artifactId>
-      <version>1.8.3</version>
-      <scope>provided</scope>
-    </dependency>
-	<dependency>
-      <groupId>net.minelink</groupId>
-      <artifactId>CombatTagPlus</artifactId>
-      <version>[1.2.0,)</version>
-      <scope>provided</scope>
-    </dependency>
-  </dependencies>
-  
-  <repositories>
-    <repository>
-      <id>civcraft-repo</id>
-      <url>http://civcraft.co:8080/plugin/repository/everything/</url>
-    </repository>
-	<repository>
-      <id>minelink-releases</id>
-      <url>http://repo.minelink.net/content/repositories/releases/</url>
-    </repository>
-  </repositories>
+	<properties>
+		<maven.compiler.source>1.6</maven.compiler.source>
+		<maven.compiler.target>1.6</maven.compiler.target>
```

Comment:
You can send pulls.  There are so many repos that need to be updated and I only remember to clean up some stuff sometimes.  Like I need to get rid of all those travis files.